### PR TITLE
Added buttons to enable and disable a link

### PR DESCRIPTION
**Pull request by Alopalao**
Issue #58
Added buttons on link_panel to enable and disable a link. Commit [914185e673b3b28cf36c994bff0f39bc29b5a38b](https://github.com/kytos-ng/topology/pull/78/commits/914185e673b3b28cf36c994bff0f39bc29b5a38b)

### CODE REVIEW

**Review by viniarck — COMMENTED** [review]
Looks good to me, left some minor threads open, let's waiting for the underlying branches to get merged first, appreciated your contribution, and it was great you managed to make progress in this feature in parallel managing other branches, well done.

File: ui/k-info-panel/link_info.kytos [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+       if(this.content === undefined) return
+       this.metadata = this.content.metadata
+     },
+     get_enabled_value() {
+       this.next_state = this.details.enabled == 'true'? 'Disable' : 'Enable'
+     },
+     msg_state_success(data){
+       let name = this.metadata.link_name !== undefined && this.metadata.link_name.length !== 0? '"' + this.metadata.link_name + '"' : this.details.id
+       let notification = {
+         title: 'Switch ' + this.next_state + 'd: Succeed',
```
[the remark this answers is not in the document]
Comment (Alopalao) [reply ui/k-info-panel/link_info.kytos]: Fixed on commit [8686ba5a4e9fd879ea73108deb0e8dc08f2ca69e](https://github.com/kytos-ng/topology/pull/78/commits/8686ba5a4e9fd879ea73108deb0e8dc08f2ca69e)

File: CHANGELOG.rst [hunk 2]
```diff
@@ -8,6 +8,8 @@ All notable changes to the ``topology`` project will be documented in this file.
 ********************************
 Added
 =====
+- Added a table of links to ``k-info-panel/switch_info``
```
[the remark this answers is not in the document]
Comment (Alopalao) [reply CHANGELOG.rst:11]: Changelog.rst updated on commit [f09eeeec244bee48a7c693d6b64753440ee52295](https://github.com/kytos-ng/topology/pull/78/commits/f09eeeec244bee48a7c693d6b64753440ee52295)

**Review by viniarck — COMMENTED** [review]
Looks good, appreciated these commits you have pushed @Alopalao, from my perspective it's just a matter of solving the conflicts now. @rmotitsuki could you add this PR to your review list and give your feedback? Thanks.

**Review by viniarck — APPROVED** [review]
Looks great to me @Alopalao, could you merge `master` into your branch? And then the team will merge this PR.

**Pull request merged**